opentech/apply/funds/management/commands/migrate_rr_applications.py

```diff
@@ -281,6 +288,7 @@ def process(self, id):
 
         try:
             submission.save()
+            submission.perform_transition(status, lead)
```

Comment:
Commenting out this line makes the migration run.

Reply:
At this very point the submission does not have a state, so `perform_transition()` throws the permission error.
In the context of migrations, setting `submission.status = self.get_workflow_state(node)` should suffice

Reply:
The original error I got was "Direct status modification is not allowed". My conclusion was that this was caused by setting "protected=True" for the FSMField. Does this sound plausible?

```
m migrate_rr_applications var/otf_export_2018-06-20/otf_rapid_response_application_export.json
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/frjo/.virtualenvs/otf-django/lib/python3.6/site-packages/django/core/management/__init__.py", line 371, in execute_from_command_line
    utility.execute()
  File "/Users/frjo/.virtualenvs/otf-django/lib/python3.6/site-packages/django/core/management/__init__.py", line 365, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/frjo/.virtualenvs/otf-django/lib/python3.6/site-packages/django/core/management/base.py", line 288, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/frjo/.virtualenvs/otf-django/lib/python3.6/site-packages/django/core/management/base.py", line 335, in execute
    output = self.handle(*args, **options)
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/contextlib.py", line 52, in inner
    return func(*args, **kwds)
  File "/Users/frjo/Sites/otf-django/opentech/apply/funds/management/commands/migrate_rr_applications.py", line 244, in handle
    self.process(id)
  File "/Users/frjo/Sites/otf-django/opentech/apply/funds/management/commands/migrate_rr_applications.py", line 262, in process
    submission.status = self.get_workflow_state(node)
  File "/Users/frjo/.virtualenvs/otf-django/lib/python3.6/site-packages/django_fsm/__init__.py", line 230, in __set__
    raise AttributeError('Direct {0} modification is not allowed'.format(self.field.name))
AttributeError: Direct status modification is not allowed
```

Reply:
Ah, I see. Yes, that is the root of the issue. There is a way to prevent this attribute error from happening, will try to get a solution back to you before the end of the day